exo-test/src/service-tester.ls

```diff
@@ -19,29 +20,30 @@ class ServiceTester extends EventEmitter
       @emit 'service-tests-skipped', @name
       return done?!
 
-    @_start-dependencies!
-    new ObservableProcess(call-args(@_create-command @service-config.tests)
-                          cwd: @config.root,
-                          env: @config
-                          stdout: {@write}
-                          stderr: {@write})
-      ..on 'ended', (exit-code) ~>
-        if exit-code > 0
-          @emit 'service-tests-failed', @name
-        else
-          @emit 'service-tests-passed', @name
-        @remove-dependencies!
-        done?(null, exit-code)
+    @_start-dependencies ~>
+      new ObservableProcess(call-args(@_create-command @service-config.tests)
+                            cwd: @config.root,
+                            env: @config
+                            stdout: {@write}
+                            stderr: {@write})
+        ..on 'ended', (exit-code) ~>
+          if exit-code > 0
+            @emit 'service-tests-failed', @name
+          else
+            @emit 'service-tests-passed', @name
+          @remove-dependencies!
+          done?(null, exit-code)
 
 
   remove-dependencies: ~>
     for dep in @service-config.dependencies or []
       DockerHelper.remove-container "test-#{dep}"
 
 
-  _start-dependencies: ~>
+  _start-dependencies: (done) ~>
     for dep in @service-config.dependencies or []
       DockerHelper.ensure-container-is-running "test-#{dep}", dep
+    wait 500, done
```

Comment:
I've created an issue to address this in the future: https://github.com/Originate/exosphere/issues/91